mscore/scoreview.cpp

```diff
@@ -1918,7 +1923,7 @@ void ScoreView::cmd(const char* s)
             if (el && (el->isNote() || el->isRest()))
                   cmdGotoElement(score()->downAlt(el));
             el = score()->selection().element();
-            while (el->isRest() && toRest(el)->isGap() && el->voice() != 3) {
```

Comment:
Making sure that `el->voice() != 3` defeats the purpose of the following few lines, which correctly terminate the _while_ loop.